azurePipelineTemplates/run_scenarios.yml

```diff
@@ -0,0 +1,117 @@
+          Write-Error "Unsupported operating system"
+          exit 1
+        }
+
+        $env:AZCOPY_AUTO_LOGIN_TYPE="AzCLI"
+
+        go build -cover -o $build_name
+
+        # Check if the executable was created
+        if (-Not (Test-Path "./$build_name")) {
+            Write-Error "Executable not found: $build_name"
+            exit 1
+        }
+        Write-Output "Executable built successfully: $build_name"
+
+        ######################################################## Scenario 1 ########################################################
+
+        # Right now we dont have this fix available in the older version of AzCopy. So, we are using the latest version of AzCopy to run the scenarios.
+        # We will update this workaround once the fix is available in the older version of AzCopy.
+
+        echo "Scenario 1: Test successful copying from BlobToPipe"
+
+        echo "Create container ${{ parameters.container_name }} under storage account ${{ parameters.storage_account_name }}"
+        & "./$build_name" "make" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}"
+
+        echo "Copy a test file from local to the created container"
+        echo "Hello, Azure!" > myfile.txt
+        & "./$build_name" "cp" "myfile.txt" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}"
+
+        & "./$build_name" "cp" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}/myfile.txt" "--from-to=BlobPipe" "--check-md5" "FailIfDifferentOrMissing" > filename
+        
+        if ($LASTEXITCODE -eq 0) {
+            Write-Output "Successfully executed the scenario."
+        } else {
+            Write-Error "AzCopy command failed with exit code $LASTEXITCODE."
+        }
+
+        # Perform cleanup
+        echo "Perform cleanup"
+        echo "Remove the test file from the container"
+        & "./$build_name" "rm" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}/myfile.txt"
+        echo "Remove the container from the storage account"
+        & az storage container delete --name ${{ parameters.container_name }} --account-name ${{ parameters.storage_account_name }}
+
+        ######################################################## Scenario 2 ########################################################
+
+        # Test copying from Object Replication Policy source container to destination container
+        # Assign an object replication policy to a storage account. 
+        # Perform azcopy copy operation to copy the blob to another container.
+
+        echo "Scenario 2: Test copying from Object Replication Policy source container to destination container"
+
+        echo "Create test file myfile.txt and copy the file to the source container srcorpcontainer under storage account azcopyorpsrcaccount having the ORP policy"
```

Comment:
Rename `srcorpcontainer` and `azcopyorpsrcaccount`

Reply:
will update